website/docs/r/mongo_cluster.html.markdown

```diff
@@ -99,10 +113,36 @@ The following arguments are supported:
 
 * `storage_size_in_gb` - (Optional) The size of the data disk space for the MongoDB Cluster.
 
+* `storage_type` - (Optional) The storage type for the MongoDB Cluster. Possible values are `PremiumSSD` and `PremiumSSDv2`. Defaults to `PremiumSSD`. Changing this forces a new resource to be created.
+
 * `tags` - (Optional) A mapping of tags to assign to the MongoDB Cluster.
 
 * `version` - (Optional) The version for the MongoDB Cluster. Possibles values are `5.0`, `6.0`, `7.0` and `8.0`.
```

Comment:
The `tags` field should always appear at the bottom of the `Arguments Reference`:

```suggestion
* `version` - (Optional) The version for the MongoDB Cluster. Possible values are `5.0`, `6.0`, `7.0` and `8.0`.

* `tags` - (Optional) A mapping of tags to assign to the MongoDB Cluster.
```

Reply:
Updated